mods/ra/rules/defaults.yaml

```diff
@@ -1091,7 +1096,8 @@
 		Categories: Decoration
 	RequiresSpecificOwners:
 		ValidOwnerNames: Neutral
-
+	Static:
+	
```

Comment:
This doesn't look right.

Reply:
Think its the diff on GH thats is messed up here!